pkg/virt-controller/watch/vm.go

```diff
@@ -2899,6 +2899,75 @@ func (c *VMController) trimDoneVolumeRequests(vm *virtv1.VirtualMachine) {
 	vm.Status.VolumeRequests = tmpVolRequests
 }
 
+func filterLiveUpdatableVolumes(lastSeenVMSpec *virtv1.VirtualMachineSpec, vm *virtv1.VirtualMachine) {
```

Comment:
not sure I get the `filter` part of this, looks like checking that all changes are to live-update-able volumes and if so  assigning assigning:

```
lastSeenVMSpec.Template.Spec.Volumes = vm.Spec.Template.Spec.Volumes
lastSeenVMSpec.Template.Spec.Domain.Devices.Disks = vm.Spec.Template.Spec.Domain.Devices.Disks
```

Reply:
Yes, this is a leftover from a previous version